Gemfile

```diff
@@ -26,6 +26,22 @@
 # See doc/COPYRIGHT.rdoc for more details.
 #++
 
+if Gem::Version.new(Bundler::VERSION) < Gem::Version.new('1.5.0')
+  abort <<-Message
+
+  *****************************************************
+  *                                                   *
+  *   OpenProject requires bundler version >= 1.5.0   *
+  *                                                   *
+  *   Please install bundler with:                    *
+  *                                                   *
+  *   gem install bundler                             *
```

Comment:
Makes sense, because if bundler was not installed at all this banner would nto show up correctly.